### DISCUSSION TITLE
Empty srt files

### DISCUSSION THREAD

**Discussion by phyfutima** [SubtitleEdit/subtitleedit#8567] [Q&A] [answered]
On Windows 10, using Purfview's Faster-Whisper model large-v3, I generated srt files from seven mp4 clips, and all seven of the resultant srt files are completely empty (and when I open the mp4 clip, it gives an error saying the associated srt file "might not be a valid subtitle file"). Why is the audio-to-text not working, am I doing something wrong?

**Comment by niksedk** [comment]
Try attaching the "whisper_log.txt" file.

Also, "large-v2" is recommended over "large-v3".

**Reply by phyfutima** [reply]
Thank you for your response, and I apologize for my delay in responding back! I got caught up in other things, but today I got back to this.

I just now updated to the latest version of Subtitle Edit (4.07) and also switched to large-v2 as you recommended and tried again. All of the resultant srt files are still completely empty. I don't know what I'm doing wrong. I've attached the whisper_log file.
[whisper_log.txt](https://github.com/user-attachments/files/16405238/whisper_log.txt)

**Reply by niksedk** [reply]
What happens if you start "whisper-faster.exe" manually?
(go to your "C:\Users\rober\AppData\Roaming\Subtitle Edit\Whisper\Purfview-Whisper-Faster" folder)

**Reply by phyfutima** [reply]
Thanks again for the response! I just tried that several times, and all that happens is a command line console opens and then closes.

**Reply by Purfview** [reply]
You need to run a command in console, read there: https://github.com/Purfview/whisper-standalone-win

**Reply by phyfutima** [reply]
I just tried:

C:\Users\rober>whisper-faster.exe "C:\Users\rober\Documents\Ranko\060124:\Clip (June 1 2024 at 1233 PM).mp4" --language English --model medium --output_dir source

And the response was:

'whisper-faster.exe' is not recognized as an internal or external command,
operable program or batch file.

Do you know what I'm doing wrong?

**Reply by Purfview** [reply]
You are trying to run it in `C:\Users\rober` folder, I doubt that exe is there.

Navigate to the right folder with `cd` command or write full path to the exe.

**Reply by phyfutima** [reply]
Thank you, it looks like we might be getting closer, but where do I find cudnn_ops_infer64_8.dll and make sure it's in my library path?

C:\Users\rober\AppData\Roaming\Subtitle Edit\Whisper\Purfview-Whisper-Faster>whisper-faster.exe "C:\Users\rober\Documents\Ranko\060124\Clip (June 1 2024 at 1233 PM).mp4" --language English --model medium --output_dir source

Standalone Faster-Whisper r192.3 running on: CUDA


Starting work on: C:\Users\rober\Documents\Ranko\060124\Clip (June 1 2024 at 1233 PM).mp4

Could not load library cudnn_ops_infer64_8.dll. Error code 126
Please make sure cudnn_ops_infer64_8.dll is in your library path!

**Reply by Purfview** [reply, marked as answer]
In Subtitle Edit there should've been a popup to download those required cuBLAS and cuDNN 8.x  for CUDA v11.x libs, didn't you see anything like that?
You can download them manually from there: https://github.com/Purfview/whisper-standalone-win/releases/tag/libs

@niksedk did you changed something about it, because the user bellow didn't had libs too?

**Reply by phyfutima** [reply]
That did it! Thank you so much!

**Reply by Whispered123** [reply]
Thanks.

**Comment by Purfview** [comment]
Try tiny model.

**Reply by phyfutima** [reply]
Thanks for the suggestion! I just tried that (both with tiny.en and tiny), and unfortunately same result: empty srt files. How can I troubleshoot this further, do you know?

**Comment by Whispered123** [comment]
I have the same problem with short video. But longer video, it shows "no text found". 
![Capture1](https://github.com/user-attachments/assets/7e2e9f33-2616-4fcc-8f05-404c10047a26)
![Capture2](https://github.com/user-attachments/assets/b464143e-19d5-4f51-b7ae-05ed23bf1bdd)
[whisper_log.txt](https://github.com/user-attachments/files/16443515/whisper_log.txt)


Model not found ? Why ?

**Reply by Purfview** [reply]
>I have the same problem with short video. But longer video, it shows "no text found".

What "same" problem?

>Model not found ? Why ?

Because you don't have model downloaded, there should be more files than "model.bin".

**Reply by Whispered123** [reply]
Thx for your reply. 
So can I download them manually ?

**Reply by Purfview** [reply]
>So can I download them manually ?

Yes -> https://huggingface.co/Systran

**Reply by niksedk** [reply]
Should look something  like this:

![image](https://github.com/user-attachments/assets/c392eff3-086b-4e85-99d5-aca23c1a2970)

![image](https://github.com/user-attachments/assets/1c603e26-fdf9-40e9-a935-d0d5148080c3)

![image](https://github.com/user-attachments/assets/863eecec-0bab-4e5b-b4b6-aa21f291eb8c)

**Reply by phyfutima** [reply]
My Purfview-Whisper-Faster folder only has the whisper-faster app and _models folder in it (no cache, _xxl_data, faster-whisper-xxl, or ffmpeg as pictured in screenshots), could that be the problem? My models folders, such as faster-whisper-tiny.en, do have the config, model, tokenizer, and vocabulary files like yours though.

**Reply by Purfview** [reply]
> could that be the problem?

No.
